src/services/users.service.ts

```diff
@@ -52,6 +52,17 @@ class UserService {
     const deleteUserData = await this.users.delete({ where: { id: userId }, select: { email: true, id: true } });
     return deleteUserData;
   }
+
+  public async manageGroups(userId: number, data: any) {
```

Comment:
If we are managing groups, why is this here, if we are managing user groups what make sense is calling it manageUserGroups to be better to read when you read it, also we should try to avoid any if we are using typescript